extensions/packageManager/src/client/Library.tsx

```diff
@@ -413,80 +559,159 @@ const Library: React.FC = () => {
-                    {strings.noComponentsInstalled}
-                  </div>
-                )}
-              </PivotItem>
-            </Pivot>
-          </Fragment>
+              ) : null}
+              {items && !items.length && !loading && (
+                <div
+                  style={{
+                    marginLeft: '50px',
+                    fontSize: 'smaller',
+                    marginTop: '20px',
+                  }}
+                >
+                  {strings.noComponentsFound}
+                </div>
+              )}
+            </Fragment>
+          )}
+
+{/* ***************************************************************************
+  *  This is the installed tab
+  ****************************************************************************/}
+
+          {(currentTab === TABS.INSTALL) && (
+            <Fragment>
+              <LibraryList
+                disabled={!ejectedRuntime}
+                install={install}
+                isInstalled={isInstalled}
+                items={installedComponents}
+                redownload={redownload}
+                removeLibrary={removeComponent}
+                updateItems={setItems}
+                onItemClick={selectItem}
+              />
+              {(!installedComponents || installedComponents.length === 0) && (
+                <div
+                 style={{
+                   marginLeft: '50px',
+                   fontSize: 'smaller',
+                   marginTop: '20px',
+                 }}
+               >
+                 {strings.noComponentsInstalled}
+               </div>
+              )}
+            </Fragment>
+          )}
+        </Stack.Item>
+
+{/* ***************************************************************************
+  *  This is the details pane
+  ****************************************************************************/}
+
+        <Stack.Item grow={0} shrink={0} disableShrink styles={{ root: { width: '400px', padding: '20px', borderLeft: '1px solid #CCC' } }}>
+          {selectedItem ? (
+            <Fragment>
+              <Stack horizontal tokens={{childrenGap:10}} >
+                <Stack.Item  grow={0} align="center" styles={{root:{width:50}}}>
+                  {selectedItem.icon ? (
+                    <img src={selectedItem.icon} width="50" height="50" alt="icon" />
+                  ): (
+                    <LetterIcon letter={selectedItem.name[0]} />
+                  )}
+                </Stack.Item>
+                <Stack.Item align="center" grow={1}>
+                  { selectedItem.authors }
+                </Stack.Item>
+                <Stack.Item align="center" grow={1} styles={{root:{textAlign:"right"}}}>
+                    <PrimaryButton onClick={install} disabled={!ejectedRuntime || !selectedItem.isCompatible} split={versionOptions!=undefined} menuProps={versionOptions}>
+                    {/* display "v1.0 installed" if installed, or "install v1.1" if not" */}
```

Comment:
remove

Reply:
I thought this might be helpful to future devs since it is a pretty complicated button